esmvalcore/preprocessor/_regrid.py

```diff
@@ -185,8 +189,45 @@ def _attempt_irregular_regridding(cube, scheme):
     return False
 
 
+def _add_regridded_aux_coords(cube, regridded_cube, target_grid):
+    """Get regridded :class:`iris.coords.AuxCoord` and corresponding dims."""
+    if regridded_cube.ndim != cube.ndim:
+        return
+    for coord in cube.coords(dim_coords=False):
+        if regridded_cube.coords(coord.name()):
+            continue
+        if coord.ndim < 2:
+            continue
```

Comment:
can you add a `logger.debug()` message here?

Reply:
The check in the first line is the check that the coordinate is not present in the regridded cube (as you requested in your comment below), the first check can probably be removed (originally I only had this and not the first check, but I see no reason why this should not work for 1D coordinates.

Reply:
aye :beer: